### DISCUSSION TITLE
French translation for joomlatools-gitpod

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomlatools/joomlatools-pages#695] [General]
Hello maybe it's not the right place to share this with you since it's a joomlatools pages discussion. But since joomlatools gitpod is also an awesome product from your team I share what I did to expand the joomlatools gitpod community to French speaking users too.
There wil be an event this friday may 21st 2021 it's a French speaking live stream  about joomla 4 testing. And I suggested them to use and try out joomlatools gitpod. Hence the quick translation in French (manually I did it since I understand both languages) and me coming to you folks and super joomlers.

Thanks again for your awesome work keep it up. 

Cheers from Martinique.
Mr Alexandre ELISÉ

https://github.com/alexandreelise/joomlatools-gitpod

**Comment by yiendos** [comment]
hey @alexandreelise thanks for the love, when we were testing your PR we notice that the joomlatools-gitpod was a little bit broke: 
* https://github.com/joomlatools/joomlatools-gitpod/issues/9
* https://github.com/joomlatools/joomlatools-gitpod/pull/10

Resulting a new release for you:
https://github.com/joomlatools/joomlatools-gitpod/tags

Open source, all about the love!
